src/crucible/aws/neptune/db_subnet_group.clj

```diff
@@ -0,0 +1,21 @@
+(ns crucible.aws.neptune.db-subnet-group
+  "Specs for AWS::Neptune::DBSubnetGroup"
+  (:require [clojure.spec.alpha :as s]
+            [crucible.aws.neptune :as neptune]
+            [crucible.encoding.keys :refer [->key]]
+            [crucible.resources :refer [spec-or-ref defresource]]))
+
+(s/def ::db-subnet-group-description (spec-or-ref string?))
+(defmethod ->key :db-subnet-group-description [_] "DBSubnetGroupDescription")
+
+(s/def ::db-subnet-group-name ::neptune/db-subnet-group-name)
+
+(s/def ::subnet-ids (spec-or-ref (s/* string?)))
```

Comment:
I remember changing a lot of (s/*) to (s/coll-of :kind vector) at some point in time, but can't remember why I did that.

Reply:
Just read up on the differences. I think `s/coll-of` is better in cases where ordering of values doesn't matter which is most of the time in CF. Will start using it.